### DISCUSSION TITLE
How can I do a equivalent query with "any"

### DISCUSSION THREAD

**Discussion by metrue** [PostgREST/postgrest#2321] [Q&A] [answered]
I'm trying to do a query like this, 
```sql
select * from posts where 'blog' = any(tags)
```
and the schema of table `posts` like this,
```sql
posts {
  id	integer Auto Increment [nextval('posts_id_seq')]
  body	text NULL
  tags	text[] NULL
}
```

I did read through the whole document, but no lucky.

**Comment by laurenceisla** [comment, marked as answer]
Currently you can't do this directly. As an alternative you could use the `cs` operator which is equivalent to `@>` in PostgreSQL (not sure about the performance difference, though). The request would be `/posts?tags=cs.{blog}` and it represents this query:

```sql
select * from posts where tags @> '{"blog"}';
```

If you want to keep using `any` you could use a `FUNCTION` with the tag as parameters, or if you use that specific filter `'blog' = any(tags)` a lot, then  a `VIEW` or a generated column could work. There's a pending enhancement related to the `any` operator here https://github.com/PostgREST/postgrest/issues/1569.
